frontend/src/js/controllers/authCtrl.js

```diff
@@ -7,11 +7,13 @@
         .module('evalai')
         .controller('AuthCtrl', AuthCtrl);
 
-    AuthCtrl.$inject = ['utilities', '$state', '$rootScope', '$timeout'];
+    AuthCtrl.$inject = ['utilities', '$state', '$rootScope', 'configService'];
```

Comment:
inject custom injectors modules in the beginning
`['utilities', 'configService', '$state', '$rootScope']`
or at end
`['$state', '$rootScope', 'utilities', 'configService']`